diff --git a/src/multimedia/multimedia.pro b/src/multimedia/multimedia.pro
index e8ea8e4..49a977e 100644
--- a/src/multimedia/multimedia.pro
+++ b/src/multimedia/multimedia.pro
@@ -15,6 +15,11 @@ contains(QT_CONFIG, opengl) | contains(QT_CONFIG, opengles2) {
 !static:DEFINES += QT_MAKEDLL
 DEFINES += QT_BUILD_MULTIMEDIA_LIB
 
+unix|win32-g++* {
+    QMAKE_PKGCONFIG_REQUIRES = QtCore QtNetwork QtGui QtWidgets
+    CONFIG += create_prl create_pc
+}
+
 load(qt_module_config)
 
 HEADERS += qtmultimediaversion.h
