diff --git a/qtmultimedia.pro b/qtmultimedia.pro
index 45af204..ee53615 100644
--- a/qtmultimedia.pro
+++ b/qtmultimedia.pro
@@ -18,5 +18,4 @@ module_qtmultimedia_tests.CONFIG = no_default_install
 !contains(QT_BUILD_PARTS,tests):module_qtmultimedia_tests.CONFIG += no_default_target
 
 SUBDIRS += module_qtmultimedia_src \
-           module_qtmultimedia_examples \
            module_qtmultimedia_tests \
